### PR TITLE
[Server Process Manager] Convert to module

### PR DESCRIPTION
**Pull request by xlecours**
Add namespace and change filename + help file

### CODE REVIEW

**Comment by MounaSafiHarab** [comment]
@xlecours 
I am working on Imaging Uploader; it is not working as I assume it depends on Server Process Manager, so I wonder if these changes will make my changes work!

**Comment by driusan** [comment]
@MounaSafiHarab What problems are you having? I don't think there should be any connection.

**Comment by MounaSafiHarab** [comment]
@driusan 

there is. Currently if you checkotu this branch, Imaging Uploader won't load (500 error) because of this:
https://github.com/aces/Loris/blob/minor/modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc#L14-L15

**Comment by xlecours** [comment]
@MounaSafiHarab It should be good now. But there will probably be conflict with yours.

**Comment by driusan** [comment]
"[Server Process Manager] Convert document repository to module"?

**Comment by MounaSafiHarab** [comment]
> "[Server Process Manager] Convert document repository to module"?

the power of copy and paste :)

**Comment by MounaSafiHarab** [comment]
> @MounaSafiHarab It should be good now. But there will probably be conflict with yours.

indeed, no more 500 errors.
I will rebase mine once yours is merged.

**Comment by driusan** [comment]
@MounaSafiHarab or @kongtiaowang can one of you test this? The code looks fine to me, but I don't have it setup..

**Comment by MounaSafiHarab** [comment]
@xlecours 

as I try to upload an image and automatically launch SPM; I get this error:
`Uncaught Error: Class 'LORIS\\server_processes_manager\\ServerProcessLauncher' not found in /var/www/loris/modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc:174`

File: modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc [hunk 1]
```diff
@@ -170,6 +171,10 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
             );
 
             if ($ImagingUploaderAutoLaunch) {
+                // Instanciate the server process module to autoload
+                // its namespace classes
+                $spm_module = \Module::factory('server_processes_manager');
```
Comment (driusan) [inline modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc]: This variable is unused and phpmd will complain.. you can just update the comment to explain that you're relying on a side-effect of the call and not assign it to anything.

**Comment by MounaSafiHarab** [comment]
@xlecours 

I can successfully now autolaunch a scan insertion.

**Pull request merged**